pages/index.vue

```diff
@@ -1,6 +1,11 @@
 <script lang="ts" setup>
```

Comment:
Vielleicht könnte man diese Redirect Logik in eine separate assets/ts Funktion auslagern, damit man sie auch auf anderen Seiten als erstes ausführen kann.